timon/webclient/webif1/src/components/Timon.vue

```diff
@@ -199,14 +203,16 @@ export default {
         rslt.state = '-'
         return rslt
       }
+      var scheduleStr = this.cfg.all_probes[probeName]['schedule'];
+      rslt.schedule = this.cfg.schedules[scheduleStr]["interval"];
```

Comment:
shouldn't this be called `rslt.sched_interval` or `rslt.interval`  as you don't store the complete schedule but just the interval value?